app-backend/common/src/main/scala/cache/CacheClient.scala

```diff
@@ -0,0 +1,70 @@
+package com.azavea.rf.common.cache
+
+import net.spy.memcached._
+
+import scala.concurrent._
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import com.azavea.rf.common.Config
+import cats.data._
+import com.typesafe.scalalogging.LazyLogging
+
+
+class CacheClient(client: => MemcachedClient)(implicit system: ActorSystem) extends LazyLogging {
+  implicit val blockingDispatcher = system.dispatchers.lookup("blocking-dispatcher")
```

Comment:
I think that we should consider defining an implicit `ExecutionContext` inside of this class:

```scala
implicit val ec: ExecutionContext =
  ExecutionContext.fromExecutor(Executors.newFixedThreadPool(16, new ThreadFactoryBuilder().setNameFormat("cache-%d").build()))
```